web/src/lib/utils/asset-utils.ts

```diff
@@ -173,3 +173,15 @@ export function getAssetRatio(asset: AssetResponseDto) {
   }
   return { width, height };
 }
+
+/**
+ * Return if an asset is an image commonly supported by web browsers
+ * @param asset Asset to check
+ * @returns true if the asset is an image commonly supported by web browsers
+ */
+export function isImageCommonlySupportedByWeb(asset: AssetResponseDto): boolean {
+  const commonWebSupportedExtensions = ['jpg', 'jpeg', 'png', 'gif', 'webp'];
+  const imgExtention = getFilenameExtension(asset.originalPath);
+
+  return commonWebSupportedExtensions.includes(imgExtention.toLowerCase());
```

Comment:
`getFilenameExtension` always returns extensions in lowercase, so no need to do `toLowerCase` here.